src/DbHelper.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Cache\Db;
+
+use Yiisoft\Db\Connection\ConnectionInterface;
+use Yiisoft\Db\Exception\Exception;
+use Yiisoft\Db\Exception\InvalidConfigException;
+
+use function explode;
+use function file_get_contents;
+use function trim;
+
+final class DbHelper
+{
+    public static function createMigration(DbCache $dbCache, bool $force = false): bool
```

Comment:
```suggestion
    public static function ensureTable(DbCache $dbCache): bool
```

I'd add `public static function `dropTable(DbCache $dbCache): bool` and remove `$force` option.

Reply:
Also we need a table name/prefix there as parameter.